src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/task/Task.java

```diff
@@ -113,10 +113,14 @@
     )
     private boolean autoAssigned;
 
-    @ApiModelProperty(required = false,
+    @ApiModelProperty(required = true,
```

Comment:
could this be a breaking change for any other team consuming our API?

Reply:
removed the required attribute